CITATION.cff

```diff
@@ -0,0 +1,32 @@
+cff-version: 1.2.0
+title: NOMAD Simulations Schema Plugin
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Nathan
+    family-names: Daelman
+    orcid: 'https://orcid.org/0000-0002-7647-1816'
+  - given-names: Bernadette
+    family-names: Mohr
+    orcid: 'https://orcid.org/0000-0003-0903-0073'
+  - given-names: José M.
+    family-names: Pizarro
+    orcid: 'https://orcid.org/0000-0002-6751-8192'
+  - given-names: Esma B.
+    family-names: Boydas
+    orcid: ''
+  - given-names: Joseph F.
+    family-names: Rudzinski
+    orcid: 'https://orcid.org/0000-0003-3403-640X'
+doi:
+repository-code: 'https://github.com/nomad-coe/nomad-simulations'
+url: 'https://nomad-coe.github.io/nomad-simulations/'
+abstract: >-
+  `nomad-simulations` is a plugin for the NOMAD software (https://zenodo.org/doi/10.5281/zenodo.8366162)
```

Comment:
"""
`nomad-simulations` is a plugin for the NOMAD software (https://zenodo.org/doi/10.5281/zenodo.8366162)
providing a common data standard and analysis tools for parsed Materials Science and Quantum Chemistry
simulations. This standard is used as a template populated by the many available parsers to transform their
inputs / outputs into a structured data format. This plugin is community-driven, and users are encourage to
contribute towards providing Findable, Accessible, Interoperable, and Reusable (FAIR) data.
"""

Let me know what you think. I wanted to put a bit more the emphases on the analysis that the normalizers provide, and not only on the RDM aspect.

Reply:
I used yours with just a very small adjustment, you can check.

Reply:
Sure, lgtm 👍🏻

Reply:
sure lgtm!